base/deprecated.jl

```diff
@@ -3246,25 +3250,22 @@ end
 
 # A[ct]_(mul|ldiv|rdiv)_B[ct][!] methods from base/linalg/givens.jl, to deprecate
 @eval Base.LinAlg begin
-    @deprecate A_mul_Bc!(A::AbstractMatrix, R::Rotation)    mul!(A, Adjoint(R))
-    @deprecate A_mul_B!(R::Rotation, A::AbstractMatrix)     mul!(R, A)
-    @deprecate A_mul_B!(G::Givens, R::Rotation)             mul!(G, R)
-    @deprecate A_mul_Bc!(A::AbstractMatrix, G::Givens)      mul!(A, Adjoint(G))
-    @deprecate A_mul_B!(G::Givens, A::AbstractVecOrMat)     mul!(G, A)
-    @deprecate A_mul_B!(G1::Givens, G2::Givens)             mul!(G1, G2)
+    @deprecate A_mul_Bc!(A::AbstractMatrix, R::Rotation)    mul1!(A, Adjoint(R))
+    @deprecate A_mul_B!(R::Rotation, A::AbstractMatrix)     mul2!(R, A)
+    @deprecate A_mul_B!(G::Givens, R::Rotation)             mul2!(G, R)
+    @deprecate A_mul_Bc!(A::AbstractMatrix, G::Givens)      mul1!(A, Adjoint(G))
+    @deprecate A_mul_B!(G::Givens, A::AbstractVecOrMat)     mul2!(G, A)
+    @deprecate A_mul_B!(G1::Givens, G2::Givens)             G1 * G2
```

Comment:
Why transform from `mul!(G1, G2)` to `G1 * G2`? :)

Reply:
The method just threw an error anyway.